### PR TITLE
refactor worker interval config

### PR DESCRIPTION
**Pull request by xcoulon**
return values in seconds rather than minutes

Fixes #ODC589

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @xcoulon for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-services/fabric8-auth/blob/4eff3b5591d9ece11229aaf0c27de5c63cc25fe5/.ike-prow/test-keeper.yaml).

File: configuration/configuration.go [hunk 1]
```diff
@@ -1120,14 +1120,14 @@ func (c *ConfigurationData) GetPostDeactivationNotificationDelayMillis() time.Du
 	return time.Duration(c.v.GetInt(varPostDeactivationNotificationDelayMillis)) * time.Millisecond
 }
 
-// GetUserDeactivationWorkerIntervalMinutes returns the interval between 2 cycles of the user deactivation worker.
-func (c *ConfigurationData) GetUserDeactivationWorkerIntervalMinutes() time.Duration {
-	return time.Duration(c.v.GetInt(varUserDeactivationWorkerIntervalMinutes)) * time.Minute
+// GetUserDeactivationWorkerIntervalSeconds returns the interval between 2 cycles of the user deactivation worker.
+func (c *ConfigurationData) GetUserDeactivationWorkerIntervalSeconds() time.Duration {
+	return time.Duration(c.v.GetInt(varUserDeactivationWorkerIntervalSeconds)) * time.Minute
```
Comment (alexeykazakov) [inline configuration/configuration.go]:
```suggestion
	return time.Duration(c.v.GetInt(varUserDeactivationWorkerIntervalSeconds)) * time.Second
```

File: configuration/configuration.go [hunk 2]
```diff
@@ -1120,14 +1120,14 @@ func (c *ConfigurationData) GetPostDeactivationNotificationDelayMillis() time.Du
...
 }
 
-// GetUserDeactivationNotificationWorkerIntervalMinutes returns the interval between 2 cycles of the user deactivation notification worker.
-func (c *ConfigurationData) GetUserDeactivationNotificationWorkerIntervalMinutes() time.Duration {
-	return time.Duration(c.v.GetInt(varUserDeactivationNotificationWorkerIntervalMinutes)) * time.Minute
+// GetUserDeactivationNotificationWorkerIntervalSeconds returns the interval between 2 cycles of the user deactivation notification worker.
+func (c *ConfigurationData) GetUserDeactivationNotificationWorkerIntervalSeconds() time.Duration {
+	return time.Duration(c.v.GetInt(varUserDeactivationNotificationWorkerIntervalSeconds)) * time.Minute
```
Comment (alexeykazakov) [inline configuration/configuration.go]:
```suggestion
	return time.Duration(c.v.GetInt(varUserDeactivationNotificationWorkerIntervalSeconds)) * time.Second
```

**Comment by xcoulon** [comment]
thanks for spotting the remaining `time.Minute` , @alexeykazakov 🙌

**Comment by xcoulon** [comment]
thanks for the review, @alexeykazakov

**Pull request merged**